nalgebra-lapack/src/svd.rs

```diff
@@ -105,7 +105,9 @@ macro_rules! svd_impl(
                 let mut lwork = -1 as i32;
                 let mut info  = 0;
                 let mut iwork = vec![0; 8 * cmp::min(nrows.value(), ncols.value())];
-
+                
+                // Using unsafe to manually ensure memory safety for external function lapack_func
+                // Rust cannot check the slices' or raw poinnters' safety
```

Comment:
Here and below, the comment must explain exactly what the safety invariants are, and how we know they're satisfied.